notebooks/Getting_started_with_Forecast-SageMaker.ipynb

```diff
@@ -0,0 +1,757 @@
+{
```

Comment:
This file is almost the same as the default file, with minor documentation updates and the clean up part. Is there anything big i am missing? In that case, you can just add it to the main notebook?